lib/i18n/backend/cache.rb

```diff
@@ -60,12 +66,26 @@ def perform_caching?
   end
 
   module Backend
-    # TODO Should the cache be cleared if new translations are stored?
     module Cache
+      @@cached_keys = []
```

Comment:
Why `cached_keys` are located in a cache module, and not in the `cache_store`? For me it seems reasonable to move `cached_keys` to the `cache_store`, to make everything consistence and interchangable. Additionally, in such case `add_cached_key` and `clear_cached_keys` will be at their right place.